xtp/include/votca/xtp/pmlocalization.h

```diff
@@ -0,0 +1,71 @@
+/*
+ *            Copyright 2009-2022 The VOTCA Development Team
+ *                       (http://www.votca.org)
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License")
+ *
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#pragma once
+#ifndef VOTCA_XTP_PMLOCALIZATION_H
+#define VOTCA_XTP_PMLOCALIZATION_H
+#include "logger.h"
+#include "votca/tools/property.h"
+#include "votca/xtp/orbitals.h"
+
+namespace votca {
+namespace xtp {
+
+class PMLocalization {
+ public:
+  PMLocalization(Logger &log, tools::Property &options) : log_(log) {
+    nrOfIterations_ = options.get(".max_iterations").as<Index>();
+    convergence_limit_ = options.get(".convergence_limit").as<double>();
+  };
+  void computePML(Orbitals &orbitals_);
```

Comment:
```suggestion
  void computePML(Orbitals &orbitals);
```

only member variables get a suffix underscore